app/src/main/java/com/bunbeauty/papakarlo/common/ui/element/card/NavigationCardWithDivider.kt

```diff
@@ -0,0 +1,75 @@
+package com.bunbeauty.papakarlo.common.ui.element.card
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Icon
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.bunbeauty.papakarlo.R
+import com.bunbeauty.papakarlo.common.ui.element.FoodDeliveryHorizontalDivider
+import com.bunbeauty.papakarlo.common.ui.element.OverflowingText
+import com.bunbeauty.papakarlo.common.ui.element.card.FoodDeliveryCardDefaults.zeroCardShape
+import com.bunbeauty.papakarlo.common.ui.icon16
+import com.bunbeauty.papakarlo.common.ui.theme.FoodDeliveryTheme
+
+@Composable
+fun NavigationCardWithDivider(
```

Comment:
Let's create common element (for filled/not filled value)
Add `value: String?` parameter and show / hide one more text